Config iter5

yuvrajzala26 opened the pull request.

yuvrajzala26 committed:
updated the config and elevator for time during moving thru floors and
load and unload when at rest.

yuvrajzala26 committed:
config updated and fixed code

AashnaNarang reviewed (changes requested):
You need to also add the timing for how the door should stay open

AashnaNarang commented on src/main/Elevator.java:
```diff
@@ -206,6 +217,12 @@ public void run() {
+			}
 		    Timer timer = new Timer("Timer");
 		    long delay = fe == null ? 500 : 
 		    	fe.getErrorCode() == 1 ? 1000 : 500;
```
The delay is basically how long the door timer is, basically change the 500 to Configurations.Time_To_Unload

yuvrajzala26 committed:
Merge branch 'main' of https://github.com/AashnaNarang/Elevator.git into config_iter5

yuvrajzala26 committed:
evrything else remained the same
file 11 and 12 both worked fine without any timed out statements

yuvrajzala26 committed:
updated the test scenarios
test scenerio 2 load balancing fails as expected
all the other test scenario passed

yuvrajzala26 committed:
loadbalancingTest now passes

AashnaNarang commented on src/main/Elevator.java:
```diff
@@ -216,9 +227,10 @@ public void run() {
-		    long delay = fe == null ? 500 : 
-		    	fe.getErrorCode() == 1 ? 1000 : 500;
+		    long delay = fe == null ? Configurations.TIME_TO_LOAD_UNLOAD: 
+		    	fe.getErrorCode() == 1 ? 1000 : Configurations.TIME_TO_LOAD_UNLOAD;
```
@krishane2018 can you double check this is correct and should we make change the 1000 into ```Configurations.TIME_TO_LOAD_UNLOAD * 2```?

yuvrajzala26 replied:
fixed it.

yuvrajzala26 committed:
Revert "loadbalancingTest now passes"

This reverts commit b1c368275c259cb880947e1cd9127b6a637517b3.

yuvrajzala26 committed:
Revert "Revert "loadbalancingTest now passes""

This reverts commit a5356f3d01d260912e6962000101952d68dcb1d9.

yuvrajzala26 committed:
Revert "Revert "Revert "loadbalancingTest now passes"""

This reverts commit d552a97d66ba439591c69e8b04299f9e84bfc591.
